docusaurus/docs/React/hooks/message-hooks.mdx

```diff
@@ -179,6 +179,26 @@ const MyCustomMessageComponent = () => {
 };
 ```
 
+### useReactionsFetcher
+
+A [custom hook](https://github.com/GetStream/stream-chat-react/blob/master/src/components/Message/hooks/useReactionsFetcher.ts) to handle loading message reactions. Returns an async function that loads and returns message reactions.
+
+```jsx
+const MyCustomMessageComponent = () => {
+  const { message } = useMessageContext();
+  const { addNotification } = useChannelActionContext();
+  const handleFetchReactions = useReactionsFetcher(message, { notify: addNotification });
+  const { data } = useQuery(['message', message.id, 'reactions'], handleFetchReactions);
```

Comment:
It is not clear where `useQuery` comes from. Maybe we could add a comment above.